homeassistant/components/unifiprotect/const.py

```diff
@@ -0,0 +1,54 @@
+"""Constant definitions for UniFi Protect Integration."""
+
+# from typing_extensions import Required
+from datetime import timedelta
+
+from pyunifiprotect.data.types import ModelType, Version
+
+DOMAIN = "unifiprotect"
+
+ATTR_EVENT_SCORE = "event_score"
+ATTR_EVENT_OBJECT = "event_object"
+ATTR_EVENT_THUMB = "event_thumbnail"
+ATTR_WIDTH = "width"
+ATTR_HEIGHT = "height"
+ATTR_FPS = "fps"
+ATTR_BITRATE = "bitrate"
+ATTR_CHANNEL_ID = "channel_id"
+
+CONF_DOORBELL_TEXT = "doorbell_text"
+CONF_DISABLE_RTSP = "disable_rtsp"
+CONF_MESSAGE = "message"
+CONF_DURATION = "duration"
+CONF_ALL_UPDATES = "all_updates"
+CONF_OVERRIDE_CHOST = "override_connection_host"
+
+CONFIG_OPTIONS = [
+    CONF_ALL_UPDATES,
+    CONF_DISABLE_RTSP,
+    CONF_OVERRIDE_CHOST,
+]
+
+DEFAULT_PORT = 443
+DEFAULT_ATTRIBUTION = "Powered by UniFi Protect Server"
+DEFAULT_BRAND = "Ubiquiti"
+DEFAULT_SCAN_INTERVAL = 2
```

Comment:
We normally don't allow scan interval below 5 seconds for polling integrations.

Reply:
This isn't actually polling since its using the websocket, but let's raise it to 5 since we don't need to check that often anyways